boards/arm/mimxrt1160_evk/mimxrt1160_evk.dtsi

```diff
@@ -33,6 +34,35 @@
 			pwms = <&flexpwm1_pwm2 0>;
 		};
 	};
+
+
+	power-states {
+		/*
+		 * RT1160 Power states are managed with set points (see page 30-33)
+		 * of the datasheet). These set points correspond to various power
+		 * savings, and associated transition (residency) times.
+		 *
+		 * Set points 1 and 10 were chosen as sane defaults to offer
+		 * limited power savings and quick transitions when entering idle for
+		 * short periods, and better power savings with longer transition
+		 * times for long idle periods
+		 */
+		idle: set_point_1_wait {
```

Comment:
note: wouldn't it be better that SoC dtsi files provide some defaults that all boards inherit by default? If needed board can always override (creating extra set-points). I don't know enough this platform to judge, though.

Reply:
Sorry for the delay in response- it's probably best to set defaults at the SOC level. I updated the PR to do that, rather than define power states at the board level.